nvflare/private/fed/server/server_aux_runner.py

```diff
@@ -93,6 +93,7 @@ def send_aux_request(self, targets: [], topic: str, request: Shareable, timeout:
             if c.token not in valid_tokens:
                 valid_tokens.append(c.token)
 
-        replies = engine.aux_send(targets=valid_tokens, topic=topic, request=request, timeout=timeout, fl_ctx=fl_ctx)
+        replies = engine.parent_aux_send(targets=valid_tokens, topic=topic, request=request,
```

Comment:
this is assuming the engine is of type "ServerEngine"

Reply:
engine = fl_ctx.get_engine(). must be "ServerEngine"